src/services/create-project.service.js

```diff
@@ -125,11 +125,18 @@ export default (
         );
 
         if (projectType === 'create-react-app') {
-          // CRA 2.0 introduces functionality to immediately init a git repo
-          // Need to do a commit after creation to make sure the repo is clean
-          childProcess.exec('git add -A && git commit -m "Add to Guppy"', {
-            cwd: projectPath,
-          });
+          try {
+            // CRA 2.0 immediately initializes a git repo upon project creation
+            // so we need to immediately commit the Guppy updates to package.json
+            childProcess.exec(
+              'git add package.json && git commit -m "Add Guppy data to package.json"',
+              {
+                cwd: projectPath,
+              }
+            );
+          } catch (err) {
+            // Ignore
```

Comment:
@joshwcomeau Is this what you meant about swallowing the errors? Or do we need to `return: false;`?

Reply:
This works!

Sorry for the delay.